### ISSUE TITLE
Angular 13 update

### ISSUE THREAD

**Issue by nbintertech** [ORNL-AMO/AMO-Tools-Desktop#4533]
update to Angular 13

**Comment by rmroot** [comment]
While you are at it can you make sure to check out all vulnerabilities that show up on npm install please

**Comment by nbintertech** [comment]
Branch is updated to ng 12 and compiling/running successfully. There are 2 loose ends:

1. Initial build time is ridiculously slow (15-20 mins)
- Need another dev to do test run for build time
- https://github.com/angular/angular-cli/issues/20792
2. There is a deprecated :lang selector somewhere in our 3rd party styles
- Also need some researching to find where this is. Possibly upgrade to bootstrap 5 or if there is a minor 4 release that removes :lang
![ng12.png](https://images.zenhubusercontent.com/5e4547eef6a311c23c81ce81/bc08694e-79c2-4829-9519-61aed9fc2d4d).

**Comment by nbintertech** [comment]
Try this again once we get MEASUR / WASM integrated

**Comment by nbintertech** [comment]
Updated to Angular 13 and can't build due to webpack errors. Branch at issue-4533-b

Problem:
- Plotly depends on node modules 'fs' and 'path', instead of native browser apis 
- Angular 11 used webpack 4 which provided polyfill support for these. Angular 13 uses webpack 5 which no longer has the polyfills

Targetting plotly 1.58 or Plotly 2.6.2 -2.9 (problem occurs on 'stream' in this v), I tried:
- adding paths to tsconfig as recommended. works for 'path' but not fs "paths": { 
      "fs": [ "./node_modules/browserify-fs" ], 
      "path": [ "./node_modules/path" ] 
    },
- adding path for browserify-fs

At this point, looking at VERIFI, maybe the MEASUR tsconfig needs a revamp. If all else fails there is a package to modify a custom webpack config with a few lines. Not ideal

Fall back solution --> the upgrade to Angular 12 built fine.
